holoviews/plotting/mpl/raster.py

```diff
@@ -14,6 +14,9 @@
 
 class RasterBasePlot(ElementPlot):
 
+    nodata = param.Integer(default=None, doc="""
+        Missing data (NaN) value for integer data""")
```

Comment:
```suggestion
        Optional missing-data value for integer data.
        If non-None, data with this value will be replaced with NaN so that it is transparent when plotted.""")
```